frontend/src/views/Question.vue

```diff
@@ -1,16 +1,16 @@
 <template>
     <section class="main-content">
       <AppBreadCrumb/>
-      <QuestionCard v-bind="question"/>
+      <QuestionCard v-bind="question" :authenticated="authenticated"/>
 
-      <AnswerForm @submitCommentForm="submitAnswer" :type="commentType" :callback="submitAnswer">
+      <AnswerForm @submitCommentForm="submitAnswer" :type="commentType" :callback="submitAnswer" :authenticated="authenticated">
```

Comment:
For the forms I think it'd be better if we replaced the whole form with text (e.g. "Login to answer this question") or ifwe greyed out/disabled the whole form

Reply:
Agree with the first part. I think the former just disappears and becomes a sentence of text.

Reply:
No problemo, but making things pretty isn't a strength of mine, so I might leave the CSS to @DarkPurple141 , but the functionality will be there.

Reply:
Actually the functionality is already there you just need to add `v-if="authenticated"` and a `v-else` :)